security-aot/build.gradle.kts

```diff
@@ -10,13 +10,9 @@ micronautBuild {
 }
 
 dependencies {
-    compileOnly(platform(mn.micronaut.core.bom))
     implementation(projects.micronautSecurityOauth2)
```

Comment:
you don't need to set platform dependencies, because you connect them with versions catalog